erica/application/EricaAuftrag/EricaAuftragService.py

```diff
@@ -14,19 +14,27 @@ class EricaAuftragServiceInterface:
     __metaclass__ = ABCMeta
 
     @abstractmethod
-    def get_status(self, auftrag_id: UUID):
+    def get_request(self, request_id: UUID):
+        pass
+
+    @abstractmethod
+    def get_all_by_skip_and_limit(self, skip: int, limit: int):
         pass
 
 
 class EricaAuftragService(EricaAuftragServiceInterface):
```

Comment:
Can we change it to EricaRequestService & EricaRequestServiceInterface & EricaRequestServiceModule?

Reply:
Oh true! Will be done!